DataMigration/src/main/java/org/sagebionetworks/dian/datamigration/tools/schedulev2/ScheduleV2Migration.java

```diff
@@ -183,31 +193,30 @@ public static TestSchedule createHMSchedule(String uId, String scheduleJson) thr
     }
 
     public static void
-        createV2Schedule(SageV1Schedule v1Schedule, String uId, String sId) throws IOException {
+    createV2Schedule(SageV1Schedule v1Schedule, String uId, String sId) throws IOException {
 
         String iANATimezone = getTimezone(v1Schedule);
         for (int i = 0; i < v1Schedule.getStudyBursts().size(); i++) {
             SageV1StudyBurst studyBurst = v1Schedule.getStudyBursts().get(i);
 
-            // V1 HM scheduling had the practice (baseline) test included and set to
-            // the first study burst start date. This shifted all study bursts by 1 day.
             // The new V2 Sage scheduling does not include the practice test, and so
             // the study as a whole, and the first study burst schedule starts.
             DateTime startDate = SageScheduleController.Companion
-                    .createDateTime(studyBurst.getStartDate()).plusDays(1);
+                    .createDateTime(studyBurst.getStartDate());
```

Comment:
We originally had the schedule on a day delay, but that has changed and the day delay is no longer needed.